Bump docs-theme-extras to v0.1.21, add pipefail to framework-tests, added descriptions (#36)
agentregistry-dev/website: content/docs/reference/cli/arctl-agent-add-prompt.md

---
title: arctl agent add-prompt

> description: "Add a prompt reference to an agent's agent.yaml file, resolved from the registry when the agent runs."

Reviewer: ```suggestion
description: "Add a prompt reference to an agent manifest."
```
